README.md

```diff
@@ -0,0 +1,63 @@
+# go-linq [![GoDoc](https://godoc.org/gopkg.in/ahmetalpbalkan/go-linq.v2?status.svg)](https://godoc.org/gopkg.in/ahmetalpbalkan/go-linq.v2) [![Build Status](https://travis-ci.org/ahmetalpbalkan/go-linq.svg?branch=v2.0)](https://travis-ci.org/ahmetalpbalkan/go-linq) [![Coverage Status](https://coveralls.io/repos/github/ahmetalpbalkan/go-linq/badge.svg?branch=v2.0)](https://coveralls.io/github/ahmetalpbalkan/go-linq?branch=v2.0)
```

Comment:
Hmm let's just remove all v2.0s here. It's not going to be a branch, I'm planning to make this live in master (and tag it as `v2.0` obviously) and archive the old version in a branch named `archive/0.9` if we need to take any fixes in the old code.

This code will be `master`, so we don't need to specify a branch name.